src/hardware/link-mso19/protocol.h

```diff
@@ -206,6 +209,7 @@ enum {
 
 /* bits - REG_DAC */
 enum {
+	DAC_DSO_VALUE_MASK =	0xfff,
 	DAC_SELECT_DSO =	0 << 15,
 	DAC_SELECT_LA =		1 << 15,
```

Comment:
is it OK that the DAC_SELECT_* enum values are not covered by DAC_DSO_VALUE_MASK ? i.e. 0xfff is 12 bits, not 15

Reply:
Yes it is correct. The value part is 12 bits and the select bit is bit15.

Reply:
ok